ui/src/notebooks/pipes/Data/TagSelectors.tsx

```diff
@@ -6,6 +6,9 @@ import {List, Gradients} from '@influxdata/clockface'
 import {PipeContext} from 'src/notebooks/context/pipe'
 import {SchemaContext} from 'src/notebooks/context/schemaProvider'
 
+// Utils
+import {event as reportEvent} from 'src/cloud/utils/reporting'
```

Comment:
i can't quite put my finger on it, but it feels like these can be one... maybe two events? like "selecting stuff" with a "is multi selection" in the tag meta data for the event